README.md

```diff
@@ -1,54 +1,91 @@
+We're making good progress (even though we're using EDD - error driven development), and it's good to see each of the errors so that when you encounter these in your real world projects you will know how to fix them. This current error is fixed by adding the following method in the static controller:
 
-And like that we've created our first rails request - it's a static request, no data is being used, no matter what, everytime we load our about page it will be exactly the same. there is nothing dynamic about it. But that teaches us the general methodology of responding to a request with rails.
+```ruby
+def about
+end
+```
 
-draw a route.
-map it to a controller action.
-program the action
-create a template to render
-render that template.
+Hitting refresh in the browser will give you a 'Template is missing' error, specifically it says: ```Missing template static/about...```
+
+We're very close to getting our view to show up, but before we make our final addition it's important to understand the difference between explicit and implicit rendering for the views. Rails gives us two options for how views are mapped between the controller and view files:
+
+* **Explicit rendering** - for explicit rendering, Rails lets you dictate what view file that you want to have the controller action mapped to.
+* **Implicit rendering** - for implicit rendering, Rails follows a standard convention that automatically looks for the view file with the same name as the controller action.
+
+First let's try out explicit rendering, create a new directory in the view's directory called ```static``` and create a new file called ```some_page.html.erb```. In that file add some basic HTML code, such as:
+
+```html
+<h1>Hello from some page</h1>
+```
+
+Inside the ```about``` method in the controller add the following code:
+
+```render "static/some_page"``` you can also use: ```render "some_page"``` (Rails will automatically look inside the view directory with the same name as the controller, but you can also give the full view path).
+
+If you refresh the ```/about``` page in the browser you will see our heading of **Hello from some page**
+
+To compare that with how Rails utilizes implicit view rendering, create a new file in the static view directory called: ```about.html.erb``` and add some HTML code such as:
+
+```html
+<h1>Hello from the about page</h1>
+```
+
+Now in the controller, remove the ```render``` call completely. If you go and refresh the browser you will now see **Hello from the about page**.
```

Comment:
It'll be a bit crazy to the students that you can have a completely empty method that does so much. I'd mention that due to a lot of implicit assumptions, this method is doing a ton!